v1/contracts/test/FixedPointTest.sol

```diff
@@ -7,19 +7,58 @@ import "../FixedPoint.sol";
 contract FixedPointTest {
     using FixedPoint for FixedPoint.Unsigned;
 
+    function wrapFromUnscaledUint(uint a) external pure returns (uint) {
+        return FixedPoint.fromUnscaledUint(a).value;
+    }
+
     function wrapAdd(uint a, uint b) external pure returns (uint) {
         return FixedPoint.Unsigned(a).add(FixedPoint.Unsigned(b)).value;
     }
 
+    // The first uint is interpreted with a scaling factor and is converted to an `Unsigned` directly.
+    function wrapMixedAdd(uint a, uint b) external pure returns (uint) {
+        return FixedPoint.Unsigned(a).add(b).value;
+    }
+
     function wrapSub(uint a, uint b) external pure returns (uint) {
         return FixedPoint.Unsigned(a).sub(FixedPoint.Unsigned(b)).value;
     }
 
+    // The first uint is interpreted with a scaling factor and is converted to an `Unsigned` directly.
+    function wrapMixedSub(uint a, uint b) external pure returns (uint) {
+        return FixedPoint.Unsigned(a).sub(b).value;
+    }
+
+    // The second uint is interpreted with a scaling factor and is converted to an `Unsigned` directly.
+    function wrapMixedSubOpposite(uint a, uint b) external pure returns (uint) {
+        return FixedPoint.sub(a, FixedPoint.Unsigned(b)).value;
```

Comment:
As we discussed IRL, maybe we should make sure the overload resolution for `uint` works when both `using SafeMath for uint` and `using FixedPoint for uint` are both present.  Can we add `using FixedPoint for uint` and change this (and other) calls to using notation like this:
```
a.sub(FixedPoint.Unsigned(b)).value;
```
?

Reply:
Done.